osu.Game/Screens/OnlinePlay/OnlinePlaySubScreen.cs

```diff
@@ -13,6 +13,8 @@ public abstract partial class OnlinePlaySubScreen : OsuScreen, IOnlinePlaySubScr
 
         public virtual string ShortTitle => Title;
 
+        protected override bool PlayExitSound => false;
```

Comment:
Inheritors also override this (should be removed for consistency):
https://github.com/ppy/osu/blob/c514550dfa0d3521b74d43aa2e0054a8b4325bdf/osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs#L47
https://github.com/ppy/osu/blob/6940579b9edb8c275de74c0f1ba4780f45577aaa/osu.Game/Screens/OnlinePlay/Lounge/LoungeSubScreen.cs#L43

This doesn't affect `OnlinePlaySongSelect`, which is in the sub screen and has the same issue when pressing home. `{Playlists|Multiplayer}ResultsScreen` is not affected because it gets pushed from the parent screen.

Reply:
> {Playlists|Multiplayer}ResultsScreen is not affected because it gets pushed from the parent screen, but may want to mute the exit sound there also?

This is already handled by the other flow change I made to `OsuScreen`.